### ISSUE TITLE
what is start_mapping?

### ISSUE THREAD

**Issue by Jasperty** [hotpotqa/hotpot#13]
hi, i have some questions about support prediction:

1. what is the meaning of start_mapping, its shape is (batch, max_para_limit, max_sent_num), why not (batch, max_sent_num)?

2. on the model of support prediction, i cannot understand these code:
 start_output = torch.matmul(start_mapping.permute(0, 2, 1).contiguous(), sp_output[:,:,self.hidden:])
 end_output = torch.matmul(end_mapping.permute(0, 2, 1).contiguous(), sp_output[:,:,:self.hidden])
 sp_output = torch.cat([start_output, end_output], dim=-1)
 sp_output_t = self.linear_sp(sp_output)
 sp_output_aux = Variable(sp_output_t.data.new(sp_output_t.size(0), sp_output_t.size(1), 1).zero_())
 predict_support = torch.cat([sp_output_aux, sp_output_t], dim=-1).contiguous()

**Comment by tomtang110** [comment]
Hi 
I have the same questions? I still do not understand what is start_mapping and end_mapping. According their shape [batch_size, paragraph_limit, max_sent_num], I would like to know what is the max_sent_num. Or what delegates do the config.sent_limit?
kind regards
Tang
